guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -13,19 +13,7 @@ include::snip_firewalld.adoc[]
 +
 [options="nowrap"]
 ----
-# firewall-cmd \
---add-port="53/udp" --add-port="53/tcp" \
---add-port="67/udp" \
---add-port="69/udp" \
---add-port="80/tcp" --add-port="443/tcp" \
-ifdef::katello,satellite,orcharhino[]
---add-port="5647/tcp" \
---add-port="8000/tcp" --add-port="9090/tcp" \
-endif::[]
-ifndef::katello,satellite,orcharhino[]
---add-port="8443/tcp" \
-endif::[]
---add-port="8140/tcp"
+# firewall-cmd --add-server=RH-Satellite-6
```

Comment:
This is Satellite-only. For orcharhino, it's "orcharhino".

Reply:
Although the new command is just a replacement of the old one and does the same thing, I think we should keep the old commands unmodified (and close this PR) or keep the new one Satellite specific. 

What's your take?

Reply:
I have not tested it completely, but your change in general looks like a great simplification/improvement. However, we should show different commands for different build targets.

For built target *satellite*, we should the command as above; for *orcharhino*, the service is called `orcharhino`; for Foreman/Katello, I don't know.

Reply:
There is no upstream version, but since it's part of firewalld itself users can use `RH-Satellite-6`. I'd recommend introducing some attribute for the firewalld profile name.

Reply:
Sounds great!

Reply:
> There is no upstream version, but since it's part of firewalld itself users can use `RH-Satellite-6`. I'd recommend introducing some attribute for the firewalld profile name.

So I should consider only `Orcharhino` (along with Satellite) at this level, right?

Reply:
You can set the attribute for all build targets to `RH-Satellite-6` and only overwrite it for orcharhino. Example: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L36 & https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc#L43

Reply:
Figuring out why attribute value is not getting rendered.

Reply:
Addressed.